src/test/system_tests/test_utils.js

```diff
@@ -737,12 +737,15 @@ async function clean_config_dir(config_fs, custom_config_dir_path) {
  * @param {nb.NativeFSContext} fs_context 
  * @param {String} file_path 
  * @param {Object} file_data 
+ * @param {{stringify_json?: Boolean}} [options={}] 
  */
-async function create_file(fs_context, file_path, file_data) {
+async function create_file(fs_context, file_path, file_data, options = {}) {
+    const buf = Buffer.from(options?.stringify_json ? JSON.stringify(file_data) : file_data);
```

Comment:
since previously we always called  stringify, I assume it the more common option. maybe we should set the default to be stringify_json = true

Reply:
I don't think so.. on a function that is called create_file() we can't assume that the default it a json file..
what I thought is to create another function that is called create_json_file() that will pass a striginified data, but I had a feeling it's redundant, WDYT?

Reply:
having it as an option is fine